main.tf

```diff
@@ -15,8 +15,7 @@ resource "aws_cloudformation_stack" "exasol_cluster" {
   capabilities = [
   "CAPABILITY_IAM"]
   on_failure   = "DELETE"
-  template_url = "https://exasol-cf-templates.s3.eu-central-1.amazonaws.com/cloudformation_template_v0.0.3.yml"
-
+  template_url = "https://exasol-cf-templates.s3.eu-central-1.amazonaws.com/cloudformation_template_v1.0.3.yml"
```

Comment:
Unlike the old link, I cannot download from this one. Extra protection, future link or broken?

Reply:
Good catch!
This is a future link. Need to get permissions for deployment first.

Reply:
:heavy_check_mark: